src/compiler/checker.ts

```diff
@@ -1448,6 +1448,7 @@ export function createTypeChecker(host: TypeCheckerHost): TypeChecker {
     var noImplicitAny = getStrictOptionValue(compilerOptions, "noImplicitAny");
     var noImplicitThis = getStrictOptionValue(compilerOptions, "noImplicitThis");
     var useUnknownInCatchVariables = getStrictOptionValue(compilerOptions, "useUnknownInCatchVariables");
+    var strictInstanceOfTypeParameters = getStrictOptionValue(compilerOptions, "strictInstanceOfTypeParameters");

```

Comment:
Let's just make this true for iterating on this PR.

```suggestion
    var strictInstanceOfTypeParameters = true;
```

Reply:
Applied this change manually with a comment. For some reason the UI wouldn't let me commit your suggestion 🤷

Reply:
I can also update the baselines to match this change temporarily if that's blocking the test suite in some way.

Reply:
If we're still only checking the extended test suite (top100, user tests, rwc, DT, perf, etc), there's no need to update the baselines; they're all separate test suites which operate independently.